lib/Runtime/Base/FunctionBody.h

```diff
@@ -1862,6 +1862,7 @@ namespace Js
         WriteBarrierPtr<void*> inlineCaches;
         InlineCachePointerArray<PolymorphicInlineCache> polymorphicInlineCaches; // Contains the latest polymorphic inline caches
         WriteBarrierPtr<PropertyId> cacheIdToPropertyIdMap;
+        WriteBarrierPtr<PropertyIdArray> formalsPropIdArray;
```

Comment:
This will increase the size of the function body, even though the function will not have eval or arguments usage. Could you check with @leirocks for alternate way to fix this?